docs/intro/reusable-apps.txt

```diff
@@ -314,6 +314,72 @@ working. We'll now fix this by installing our new ``django-polls`` package.
 
 #. Run the development server to confirm the project continues to work.
 
+Testing your app
+================
+
+Since we moved the ``polls`` directory out of the project,
+we don't have the project's ``settings.py`` and ``urls.py`` anymore.
+We need another way to configure our app in order to test it.
+
+#. First create a ``tests`` directory and move the tests over from the
+   ``polls`` directory. Because the tests now reside in a different module
+   the relative import ``from .models import Poll`` has to be changed to
+   ``from polls.models import Poll``.
```

Comment:
Also note @RaphaelKimmig that all lines should be wrapped at 79cols, so please check this and all the following changes to match this style:
```suggestion
Since we moved the ``polls`` directory out of the project, we don't have the
project's ``settings.py`` and ``urls.py`` anymore. We need another way to
configure our app in order to test it.

#. First create a ``tests`` directory and move the tests over from the
   ``polls`` directory. Because the tests now reside in a different module the
   relative import ``from .models import Poll`` has to be changed to
   ``from polls.models import Poll``.
```

Reply:
Alright, reformatted :-)